config/crd/bases/ndb.nutanix.com_databases.yaml

```diff
@@ -25,11 +25,8 @@ spec:
     - jsonPath: .status.status
       name: Status
       type: string
-    - jsonPath: .status.id
-      name: Database Instance ID
-      type: string
-    - jsonPath: .status.dbServerId
-      name: Database Server ID
+    - jsonPath: .status.type
+      name: Type
```

Comment:
Were these changes made by `make generate` automatically?

Reply:
yes, "make manifests" to be precise.